doc/nrf/libraries/nrf_rpc/nrf_rpc_utils.rst

```diff
@@ -0,0 +1,17 @@
+.. _nrf_rpc_dev_info:
+
+nRF RPC utilities
+#################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This library allows you to obtain the device information functions of the :ref:`nrfxlib:nrf_rpc` on a remote server.
+
+API documentation
+*****************
+
+| Header file: :file:`include/nrf_rpc/nrf_rpc_dev_info.h`
+
+.. doxygengroup:: nrf_rpc_dev_info
```

Comment:
```suggestion
.. doxygengroup:: nrf_rpc_utils
```
I believe this broke the doc build.

Reply:
@peknis The header file declaring the doxygen group was located in an incorrect folder. now, the build should be fine.